features/em.yml

```
@@ -1,7 +1,6 @@
-draft_date: 2024-06-05
 name: <em>
-description: The `<em>` element TODO.
+description: The `<em>` element element makes text italic in order to identify idomatic content.
```

It would be good to use "emphasis" or "emphasize" in the description for searchability. I'll also suggest descriptions that put the styling effect first for `<b>`, `<i>`, and `<u>`, but the semantics first for `<em>` and `<strong>`.

So in this case:

```suggestion
description: The `<em>` HTML element emphasizes or stresses the content, rendering text as italic by default.
```

---

Done for all.